src/main/java/com/shank/offcoder/cf/DownloadManager.java

```diff
@@ -24,6 +24,8 @@
 import java.util.List;
 
 public class DownloadManager {
+    private static final int pageLimit = 25; // Threshold for number of downloaded questions displayed per page.
```

Comment:
@Aqua-16 It's common to have final constants named in CAPS.

`private static final int PAGE_LIMIT = 25;`

Reply:
Understood sir